### ISSUE TITLE
Blank page

### ISSUE THREAD

**Issue by rnenjoy** [flipace/lovli.js#8]
Hello,

I have installed lovli.js on my LAN server 192.168.0.2

Browsing to it :3000 gives me a blank Hello world page with:

```
<!doctype html>
    <html>
      <head>
        <title>Hello World</title>
        <link rel="stylesheet" type="text/css" href="http://192.168.0.2:9095/static/styles.css" />
      </head>
      <body>
        <div id='root'></div>
        <script src="http://192.168.0.2:9095/static/client.bundle.js"></script>
      </body>
    </html>
```

The style and the client.bundle is loaded but then nothing. Any idea what i have done wrong? BTW it says in the FAQ that hot module replacement isnt activated. But why does it say the module is enabled everywhere?

**Comment by deanhamilton11** [comment]
Exact same thing happening here, the horizon client isn't connecting for me.

I think its some sort of problem with the dependencies, I cloned this repo yesterday morning and that copy still works, recloned the same version last night and now the client doesn't connect.

After running npm install in the first version i cloned, it no longer works as well. No idea where to go from here.

**Comment by flipace** [comment]
This is weird. `.connect` won't even try to connect...

**Comment by flipace** [comment]
Alright, I found the issue - a temporary workaround is to specifically install rxjs@5.0.0-beta.7 - a new version ( beta8 ) has been published yesterday and contains changes which break the horizon client.

I'm going to do a PR for the horizon client. Until the new release is published, please use rxjs@5.0.0-beta.7 as a workaround.

See: https://github.com/rethinkdb/horizon/issues/484

**Comment by flipace** [comment]
Temporarily fixed with: a4abf8a5c54538a6e69979385d57e646060101a1
